docs/requirements.txt

```diff
@@ -6,3 +6,4 @@ sphinxcontrib-programoutput==0.16
 git+https://github.com/pytorch/pytorch_sphinx_theme.git#egg=pytorch_sphinx_theme
 torch>=1.6.0
 numpy>=1.19.5
+pyre-extensions == 0.0.23
```

Comment:
this was the reason why the deployment job was dying I believe, given the log. Works fine locally with that (see screenshot below)